parsl/providers/kubernetes/kube.py

```diff
@@ -110,31 +110,29 @@ def submit(self, cmd_string, blocksize, tasks_per_node, job_name="parsl"):
              - None: At capacity, cannot provision more
              - job_id: (string) Identifier for the job
         """
-        if not self.resources:
-            cur_timestamp = str(time.time() * 1000).split(".")[0]
-            job_name = "{0}-{1}".format(job_name, cur_timestamp)
-
-            if not self.deployment_name:
-                deployment_name = '{}-deployment'.format(job_name)
-            else:
-                deployment_name = '{}-{}-deployment'.format(self.deployment_name,
-                                                            cur_timestamp)
-
-            formatted_cmd = template_string.format(command=cmd_string,
-                                                   worker_init=self.worker_init)
-
-            self.deployment_obj = self._create_deployment_object(job_name,
```

Comment:
_create_deployment and _create_deployment_object are dead code that should be removed maybe now.

Reply:
removed